docs/installation.rst

```diff
@@ -27,6 +27,14 @@ For example:
     python -m pip install --upgrade pip
 
 
+Unsupported platforms
+---------------------------------------------------------
+
+On these platforms, wheels are not provided and PyMuPDF fails to build:
```

Comment:
Pedantic - PyMuPDF - > |PyMuPDF| , it just means that it will cite our product nicely (see header.rst for code)

Reply:
Ah yes, fixed in my tree.